ivy_tests/test_ivy/test_frontends/test_torch/test_utilities.py

```diff
@@ -0,0 +1,73 @@
+# global
+# import ivy
+import numpy as np
+from hypothesis import given, strategies as st
+
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.torch as ivy_torch
+
+
+# noinspection DuplicatedCode
+@st.composite
+def _arrays_idx_n_dtypes(draw):
```

Comment:
Is there any reason that you have included this helper function? It comes from test_manipulation, and it doesn't seem to have anything to do with the function you are testing.

Reply:
According to ivy frontend video I assumed I was supposed to add it in my function.

Reply:
Am I supposed to remove whole code from line no. 13 to 43 , or do some changes?

Reply:
This specific function is used by the concat function to generate data that works for it. You should make use of the dtypes_and_values function in helpers.py to generate your arrays. Have a look at the helper function, and the documentation on pytorch for `result_type` to figure out what kind of data you need to generate